packages/next/server/next-server.ts

```diff
@@ -98,6 +101,7 @@ import ResponseCache, {
 import { NextConfigComplete } from './config-shared'
 import { parseNextUrl } from '../shared/lib/router/utils/parse-next-url'
 import isError from '../lib/is-error'
+import { Worker } from '../lib/worker'
```

Comment:
Out of curiosity, why not use [Worker threads](https://nodejs.org/api/worker_threads.html) directly?

Reply:
Just because we already have jest-worker in several places (e.g. build), no specific reason. Plan to switch to worker threads later so we can do postMessage easier.